packages/components/modules/payments/web/PaymentMethodsManagementComponent/components/PaymentMethodsItem.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'react'
+
+import { MoreVert } from '@mui/icons-material'
+import { Box, Chip, Divider, IconButton, Typography } from '@mui/material'
+
+import { getCardIcon } from '../../CheckoutComponent/utils'
```

Comment:
lets move `getCardIcon` to `packages/components/modules/payments/web/utils/stripe.ts` or `.../utils.ts`, so we avoid using utils that are scoped only for a component (e.g `CheckoutComponent`) and make it available for other components on different hierarchy levels